Enable quantized cat

mcremon-meta opened the pull request:
Summary: As titled. Use it only in the WakeWord quantizer for now, because it has implications on the numerics in general.

Reviewed By: zonglinpeng

Differential Revision: D69499329

mcremon-meta commented:
@dbort @mergennachin @tarun292  have you seen linter errors like this one? It doesn't seem suggest a change, and linting on my own clone of ET (native mac) is clean. Not sure how to proceed.

mergennachin commented:
@mcremon-meta when you run locally, do you have the same lintrunner version as the CI? 

https://github.com/pytorch/executorch/blob/main/CONTRIBUTING.md#lintrunner

mcremon-meta commented:
> @mcremon-meta when you run locally, do you have the same lintrunner version as the CI?
> 
> https://github.com/pytorch/executorch/blob/main/CONTRIBUTING.md#lintrunner

yes I did those exact steps this morning, so should be the same

mcremon-meta commented:
might have been trailing spaces, let's see if new version is happy. If it's that it'd be nice if the linter message could say that, since the spaces don't show on the diff view (even when I highlight)

mcremon-meta committed:
Enable quantized cat (#8757)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/8757

As titled. Use it only in the WakeWord quantizer for now, because it has implications on the numerics in general.

Reviewed By: tarun292, zonglinpeng

Differential Revision: D69499329
